inference/python/infbench/cutlass.py

```diff
@@ -27,12 +81,56 @@ def getConstants(modelDir):
         return []
 
 
-class sgemm(sgemmBase):
+class cutlassSgemm(sgemmBase):
+    def __init__(self, modelArgs):
+        self.M = 10000
+        self.N = 8000
+        self.K = 10000
+        self.alpha = 1
+        self.beta = 0
+        self.modelDir = modelArgs
+
     def run(self, dat):
         """Run the model against input 'dat'. Dat is expected to be a bytes
        object that can be converted to numpy/tvm and passed to the model as
        input."""
-        pass
+        import pycuda.autoinit  # NOQA
```

Comment:
This is slightly dodgy. I think it more or less works out on ray, but it can lead to some unexpected errors. It's problematic in local mode where we run multiple copies of the same model in one process (though mlperf mode in local is broken right now anyway).

Anyway, a safer way is to mimic python/infbench/testModel.py:testModelNative where we manually create a cuda context and destroy it when we're done.